website/auth.py

```diff
@@ -314,6 +314,17 @@ def send_email_template(template, email, link=None, username=gettext('user')):
 
     send_email(email, subject, body_plain, body_html)
 
+
+# By default, emails are sent in the locale of the logged-in user.
+# This function is to be used if the email needs to be sent in another locale.
+def send_localized_email_template(locale, template, email, link=None, username=None):
+    with force_locale(locale):
+        if username is None:
+            # We want to use the correct locale for this text
```

Comment:
Do we mean "current" here? (in stead of correct?)

Reply:
Could be either, I guess. We want the locale that's used to send the email, so the correct locale. And at this point it's also the current locale. It's not the correct locale when you enter the function, so we can't define it as username=gettext('user') (as is the case in send_email_template() )

Reply:
I think I don't understand in what scenario we do not have a username, which is why the comment it hard to understand (but it is ok, I understand what it does)